### PR TITLE
Visitor can see ads

### PR DESCRIPTION
**Pull request by Ehugo2000**
[Visitor can see ads CLIENT](https://www.pivotaltracker.com/story/show/176031746)
### User Story
```
As a Visitor,
In order to get great offers,
I would like to see ads
```
## Changes proposed in this pull request:
* Adding ads to index page
* Adding overall styling
* Adding favicon
* Updating translations
## What I have learned working on this feature:
* Add favicons
* Semantic UI images
## Packages/Gems added
* 
## Screenshots (Client)
Screenshot from 2021-01-17 16-49-22

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -10,18 +10,13 @@ const ArticleCard = ({ article }) => {
       as={NavLink}
       to={`/articles/${article.id}`}
     >
-      <Item.Image
-        size="small"
-        wrapped
-        ui={false}
-        src={article.image}
-      />
+      <Item.Image size='small' wrapped ui={false} src={article.image} />
```
Comment (KyunginNa) [inline src/components/ArticleCard.jsx:13]: Let's display category here too

**Review by KyunginNa — COMMENTED** [review]
Awesome job!

File: src/components/ArticleCard.jsx (hunk 1 above)
Comment (Ehugo2000) [reply src/components/ArticleCard.jsx:13]: Done!

**Comment by KyunginNa** [comment]
AWESOME ADS

**Pull request merged**